### PR TITLE
Namespace change

### PR DESCRIPTION
**Pull request by superdiana**

### CODE REVIEW

File: voice/join-outbound-calls.py [hunk 1]
```diff
@@ -3,27 +3,23 @@
 
 app = Flask(__name__)
 
+
```
Comment (slorello89) [inline voice/join-outbound-calls.py:6]: Missing where the variables here come from - supposed to edit ahead of time? Maybe we could use dotenv? :)

File: voice/make-outbound-call-ncco.py [hunk 2]
```diff
@@ -1,16 +1,26 @@
 #!/usr/bin/env python3
-import nexmo
+import vonage, os
 from pprint import pprint
```
Comment (slorello89) [inline voice/make-outbound-call-ncco.py:4]: missing dotenv load

File: voice/play-audio-stream-into-call.py [hunk 3]
```diff
@@ -1,9 +1,14 @@
-import nexmo
+import vonage, os
```
Comment (slorello89) [inline voice/play-audio-stream-into-call.py:2]: missing dotenv load

File: voice/play-dtmf-into-call.py [hunk 4]
```diff
@@ -1,8 +1,13 @@
-import nexmo
+import vonage, os
```
Comment (slorello89) [inline voice/play-dtmf-into-call.py:2]: missing dotenv load

File: voice/play-tts-into-call.py [hunk 5]
```diff
@@ -1,9 +1,13 @@
-import nexmo
+import vonage, os
```
Comment (slorello89) [inline voice/play-tts-into-call.py:2]: missing dotenv load

File: voice/record-a-call-with-split-audio.py [hunk 6]
```diff
@@ -4,38 +4,36 @@
```
Comment (slorello89) [inline voice/record-a-call-with-split-audio.py:4]: Missing where the variables are meant to be added here - I'm thinking have env variables would be preferable?

File: voice/record-a-call-with-split-audio.py [hunk 7]
```diff
@@ -4,38 +4,36 @@
...
 app = Flask(__name__)
 
+
 @app.route("/webhooks/answer")
 def answer_call():
     ncco = [
         {
             "action": "talk",
-            "text": "Hi, we will shortly forward your call. This call is recorded for quality assurance purposes."
+            "text": "Hi, we will shortly forward your call. This call is recorded for quality assurance purposes.",
```
Comment (slorello89) [inline voice/record-a-call-with-split-audio.py:13]: extra comma at the end here

File: voice/record-a-call.py [hunk 8]
```diff
@@ -4,36 +4,31 @@
...
-@app.route("/webhooks/recordings", methods=['POST'])
+
+@app.route("/webhooks/recordings", methods=["POST"])
 def recordings():
     data = request.get_json()
     pprint(data)
     return "webhook received"
 
-if __name__ == '__main__':
+
```
Comment (slorello89) [inline voice/record-a-call.py:32]: remove extra whitespace here and on line 25

File: voice/retrieve-info-for-a-call.py [hunk 9]
```diff
@@ -1,12 +1,15 @@
 #!/usr/bin/env python3
-import nexmo
+import os
+from vonage import Voice, Client
 from pprint import pprint
```
Comment (slorello89) [inline voice/retrieve-info-for-a-call.py:5]: missing dotenv load

File: voice/retrieve-info-for-all-calls.py [hunk 10]
```diff
@@ -1,9 +1,12 @@
 #!/usr/bin/env python3
-import nexmo
+import vonage, os
```
Comment (slorello89) [inline voice/retrieve-info-for-all-calls.py:2]: missing dotenv load

File: voice/transfer-a-call.py [hunk 11]
```diff
@@ -1,12 +1,17 @@
 #!/usr/bin/env python3
-import nexmo
+import vonage, os
 from pprint import pprint
```
Comment (slorello89) [inline voice/transfer-a-call.py:4]: missing dotenv load

File: voice/transfer-call-inline-ncco.py [hunk 12]
```diff
@@ -1,30 +1,45 @@
 # Import dependencies
-import nexmo
+import vonage
 import os
 from os.path import join, dirname
 from dotenv import load_dotenv
 
 # Load the environment
-envpath = join(dirname(__file__), './.env')
+envpath = join(dirname(__file__), "./.env")
```
Comment (slorello89) [inline voice/transfer-call-inline-ncco.py:8]: missing `.` in path

File: application/delete-application.py [hunk 13]
```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/env python3
-import nexmo
+import vonage, os
```
Comment (slorello89) [inline application/delete-application.py:2]: missing dotnev load

File: application/get-application.py [hunk 14]
```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/env python3
-import nexmo
+import vonage, os
```
Comment (slorello89) [inline application/get-application.py:2]: missing dotnev load

File: application/list-applications.py [hunk 15]
```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env python3
-import nexmo
+import vonage, os
```
Comment (slorello89) [inline application/list-applications.py:2]: missing dotnev load

File: application/update-application.py [hunk 16]
```diff
@@ -1,49 +1,52 @@
 #!/usr/bin/env python3
-import nexmo
+import vonage, os
 from pprint import pprint
```
Comment (slorello89) [inline application/update-application.py:3]: missing dotnev load

File: initialize/application.py [hunk 17]
```diff
@@ -1,6 +1,9 @@
-import nexmo
+import vonage, os
```
Comment (slorello89) [inline initialize/application.py:2]: missing dotnev load

File: initialize/basic.py [hunk 18]
```diff
@@ -1,3 +1,6 @@
-import nexmo
+import vonage, os
```
Comment (slorello89) [inline initialize/basic.py:2]: missing dotnev load

File: initialize/full.py [hunk 19]
```diff
@@ -1,8 +1,13 @@
-import nexmo
+import vonage, os
```
Comment (slorello89) [inline initialize/full.py:2]: missing dotnev load

**Review by slorello89 — CHANGES_REQUESTED** [review]
Few items:
1. We have a somewhat inconsistent method of loading environment vars into the scripts. I think our best bet would be to have everything use dotenv - I've made notes where the dotenv's path was off, or the load was missing altogether.
2. I think there's some missing updates from the README - definitely many references to nexmo in there.
3. Some trivial comments requesting minor formatting changes.

**Comment by slorello89** [comment]
also, requirements.txt needs to point at the vonage package.

File: application/create-application.py [hunk 20]
```diff
@@ -1,28 +1,30 @@
 #!/usr/bin/env python3
-import nexmo
+import vonage, os
 from pprint import pprint
```
[the remark this answers is not in the document]
Comment (dragonmantank) [reply application/create-application.py:4]: That sound suspiciously like a good first issue for someone :smile: 

In general standalone scripts should pull from the root of the problem, full apps (like ones testing webhooks) should have their own .env files as those types of examples should be self-contained.

**Review by slorello89 — COMMENTED** [review]
I've run everything, everything looks good

**Review by slorello89 — APPROVED** [review]
LGTM

**Pull request merged**